src/Congo.WebApi/Data/CongoContext.cs

```diff
@@ -1,13 +1,22 @@
-﻿using Congo.WebApi.Data.Models;
+﻿using Congo.WebApi.Data.Configurations;
+using Congo.WebApi.Data.Models;
 using Microsoft.EntityFrameworkCore;
 
 namespace Congo.WebApi.Data
 {
     public class CongoContext : DbContext
     {
-        public DbSet<Product> Products { get; set; }
         public CongoContext(DbContextOptions<CongoContext> options) : base(options)
         {
         }
+
+        public DbSet<Product> Products { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            new ProductEntityTypeConfiguration().Configure(modelBuilder.Entity<Product>());
```

Comment:
Can we use the assembly scanning version instead here? That way we don't need to explicitly add another entry for each configuration we set up.

```csharp
modelBuilder.ApplyConfigurationsFromAssembly(typeof(CongoContext).Assembly);
```